tools/functional-tester/etcd-runner/main.go

```diff
@@ -43,6 +45,7 @@ func main() {
 	endpointStr := flag.String("endpoints", "localhost:2379", "endpoints of etcd cluster")
```

Comment:
@xiang90 but for raising them as another PR this needs to be merged first. Or shall i add them here as another commit with different commit title?